packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -131,7 +132,7 @@ export function b58decodeL2Address(payload: string) {
  */
 export function encodePubKey(value: string) {
   if (value.substring(0, 2) === '00') {
-    const pref: { [key: string]: Uint8Array } = {
+    const pref: { [key: string]: Uint8Array; } = {
```

Comment:
Am curious are these semi-colons in curly brackets from linter? Would like to know why it only pops up now?

Reply:
from my formatter 
types want to be ended with `;`  even for values of keys in objects